chart/values.yaml

```diff
@@ -321,6 +323,23 @@ backfill:
   tolerations: []
 
 
+deleteIndexes:
+  enabled: false
+  log:
+    level: "info"
+  action: "delete-indexes"
+  error_codes_to_retry: ""
+  schedule: "*/10 * * * *"
```

Comment:
also: it can be a lot less frequent, to reduce IO. Maybe once a day

Reply:
I changed it to run once a day https://github.com/huggingface/datasets-server/pull/1536/commits/f38ba31e3b7664ee3322416f71f07444cfddf906